File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/Models/ReservationItem.cs
using System;

namespace ConferenceManagement
{
    public class ReservationItem
    {
        public ReservationItem(Guid seatTypeId, int quantity)
        {
            SeatTypeId = seatTypeId;
            Quantity = quantity;
        }

        public Guid SeatTypeId { get; private set; }
        public int Quantity { get; private set; }
    }
}

File: samples/conference/management-bc/src/main/java/com/microsoft/conference/management/domain/Models/ReservationItem.java
package com.microsoft.conference.management.domain.Models;

public class ReservationItem {
    public String SeatTypeId;
    public int Quantity;
    public ReservationItem(String seatTypeId, int quantity) {
        SeatTypeId = seatTypeId;
        Quantity = quantity;
    }
}
